flink-core/src/main/java/org/apache/flink/configuration/HistoryServerOptions.java

```diff
@@ -89,6 +89,16 @@
 			.withDescription("Enable HTTPs access to the HistoryServer web frontend. This is applicable only when the" +
 				" global SSL flag security.ssl.enabled is set to true.");
 
+	/**
+	 * Specifies whether the HistoryServer should process job archive deletion. If this option is
```

Comment:
I'd omit the first sentence, `process job archive deletion` is quite broad, and only really makes _sense_ after the second sentence.

Reply:
First sentence removed. Thank you.